RecoTracker/MkFit/test/dumpMkFitGeometryPhase2.py

```diff
@@ -4,23 +4,23 @@
 from Configuration.ProcessModifiers.trackingMkFitCommon_cff import trackingMkFitCommon
 trackingMkFit = cms.ModifierChain(trackingMkFitCommon)
 
-###################################################################
-# Set default phase-2 settings
-###################################################################
-import Configuration.Geometry.defaultPhase2ConditionsEra_cff as _settings
-_PH2_GLOBAL_TAG, _PH2_ERA = _settings.get_era_and_conditions(_settings.DEFAULT_VERSION)
-
-process = cms.Process('DUMP', _PH2_ERA, trackingMkFit)
```

Comment:
Don't undo this, this is done precisely to avoid chasing the latest geometry when the old ones are removed.

Reply:
Thank you! So I put this back as-is and remove the `process = ...` on the following line? And, I assume, the same for the `process.GlobalTag = ...` change below.

We still have explicit load of `GeometryExtended2026D110Reco_cff`. Is this OK? We do need to have control over this in some way or another.

Reply:
I think that if you keep it as it is it should still work.

> We still have explicit load of GeometryExtended2026D110Reco_cff. Is this OK? 

`GeometryExtended2026DefaultReco_cff` currently points to that indeed:

https://github.com/cms-sw/cmssw/blob/ecaad523d223d07de8024e9ed3a09b3b7dfba507/Configuration/Geometry/python/GeometryExtended2026DefaultReco_cff.py#L4

and

https://github.com/cms-sw/cmssw/blob/ecaad523d223d07de8024e9ed3a09b3b7dfba507/Configuration/Geometry/python/defaultPhase2ConditionsEra_cff.py#L5

so the way it is, is guaranteed to stay in sync with the default geometry. 

> We do need to have control over this in some way or another.

do you mean you want it to be configurable from command line? Would parsing an argument with `argparse` work here?

Reply:
We have restored this in [bfab41c](https://github.com/cms-sw/cmssw/pull/46325/commits/bfab41c258f0620bb57443bc6c4e7bba5e3262b4).